<ui version="4.0" >
 <class>PlacemarkInfoDialog</class>
 <widget class="QDialog" name="PlacemarkInfoDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Marble Infocenter</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QTabWidget" name="m_pTabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Data Sheet</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QGridLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item row="0" column="0" >
          <widget class="QLabel" name="name_val_lbl" >
           <property name="toolTip" >
            <string>Name of the location</string>
           </property>
           <property name="whatsThis" >
            <string>Name of the location</string>
           </property>
           <property name="text" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:14pt; font-weight:600;">Name&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="textInteractionFlags" >
            <set>Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>16</height>
            </size>
           </property>
          </spacer>
         </item>
         <item rowspan="4" row="0" column="1" >
          <widget class="QLabel" name="flag_val_lbl" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>120</width>
             <height>90</height>
            </size>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0" >
          <widget class="QLabel" name="role_val_lbl" >
           <property name="toolTip" >
            <string>Role</string>
           </property>
           <property name="whatsThis" >
            <string>Role of the location.</string>
           </property>
           <property name="text" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-style:italic;">Role&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="textInteractionFlags" >
            <set>Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="altername_val_lbl" >
           <property name="toolTip" >
            <string>Alternative name</string>
           </property>
           <property name="whatsThis" >
            <string>Alternative name of the location.</string>
           </property>
           <property name="text" >
            <string>Alternative Name</string>
           </property>
           <property name="textInteractionFlags" >
            <set>Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="description_lbl" >
         <property name="text" >
          <string>Short Description:</string>
         </property>
         <property name="buddy" >
          <cstring>description_val_browser</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="description_val_browser" >
         <property name="enabled" >
          <bool>false</bool>
         </property>
         <property name="toolTip" >
          <string>Description</string>
         </property>
         <property name="whatsThis" >
          <string>Description field</string>
         </property>
         <property name="html" >
          <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;">No description available.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:9pt;">If you are connected to the Internet please click the "Wikipedia" tab at the top of this page to get more information.&lt;/p>&lt;/body>&lt;/html></string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>330</width>
           <height>16</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="frame" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="coordinates_lbl" >
            <property name="text" >
             <string>Coordinates:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="textInteractionFlags" >
             <set>Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLabel" name="coordinates_val_lbl" >
            <property name="text" >
             <string>000° 00' 00" N
000° 00' 00" E </string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="textInteractionFlags" >
             <set>Qt::TextEditable</set>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="elevation_lbl" >
            <property name="text" >
             <string>Elevation:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="textInteractionFlags" >
             <set>Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2" >
           <widget class="QLabel" name="elevation_val_lbl" >
            <property name="text" >
             <string>9000.0 m</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="textInteractionFlags" >
             <set>Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="area_lbl" >
            <property name="text" >
             <string>Area:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="textInteractionFlags" >
             <set>Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1" colspan="2" >
           <widget class="QLabel" name="area_val_lbl" >
            <property name="text" >
             <string>0.0 sq km</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="textInteractionFlags" >
             <set>Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0" >
           <widget class="QLabel" name="country_lbl" >
            <property name="text" >
             <string>Country:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="textInteractionFlags" >
             <set>Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1" colspan="2" >
           <widget class="QLabel" name="country_val_lbl" >
            <property name="text" >
             <string>Country</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="textInteractionFlags" >
             <set>Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item row="4" column="0" >
           <widget class="QLabel" name="population_lbl" >
            <property name="text" >
             <string>Population:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="textInteractionFlags" >
             <set>Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item row="4" column="1" colspan="2" >
           <widget class="QLabel" name="population_val_lbl" >
            <property name="text" >
             <string>1000000</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="textInteractionFlags" >
             <set>Qt::LinksAccessibleByMouse</set>
            </property>
           </widget>
          </item>
          <item row="5" column="0" >
           <widget class="QLabel" name="diameter_lbl" >
            <property name="text" >
             <string>Diameter:</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="textInteractionFlags" >
             <set>Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item row="5" column="1" >
           <widget class="QLabel" name="diameter_val_lbl" >
            <property name="text" >
             <string>TextLabel</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Wikipedia</string>
      </attribute>
      <attribute name="icon" >
       <iconset resource="../marble.qrc" >
        <normaloff>:/icons/wikipedia.png</normaloff>:/icons/wikipedia.png</iconset>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="margin" >
        <number>9</number>
       </property>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="m_pBackButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="toolTip" >
            <string>Back</string>
           </property>
           <property name="whatsThis" >
            <string>Move backwards one step in the browsing history.</string>
           </property>
           <property name="text" >
            <string>Back</string>
           </property>
           <property name="icon" >
            <iconset>
             <normaloff>../../../../../../../../.designer/backup</normaloff>../../../../../../../../.designer/backup</iconset>
           </property>
           <property name="toolButtonStyle" >
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_pForwardButton" >
           <property name="enabled" >
            <bool>false</bool>
           </property>
           <property name="toolTip" >
            <string>Forward</string>
           </property>
           <property name="whatsThis" >
            <string>Move forward one step in the browsing history.</string>
           </property>
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset>
             <normaloff>../../../../../../../../.designer/backup</normaloff>../../../../../../../../.designer/backup</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_pPrintButton" >
           <property name="text" >
            <string>Print Page</string>
           </property>
           <property name="icon" >
            <iconset resource="../marble.qrc" >
             <normaloff>:/icons/document-print.png</normaloff>:/icons/document-print.png</iconset>
           </property>
           <property name="toolButtonStyle" >
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>281</width>
             <height>31</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="TinyWebBrowser" name="m_pWikipediaBrowser" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_pStatusLabel" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="m_pLine" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_pButtonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TinyWebBrowser</class>
   <extends>QTextBrowser</extends>
   <header>TinyWebBrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../marble.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>m_pButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>PlacemarkInfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>266</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>PlacemarkInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>334</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pBackButton</sender>
   <signal>clicked()</signal>
   <receiver>m_pWikipediaBrowser</receiver>
   <slot>backward()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>54</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel" >
     <x>55</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pForwardButton</sender>
   <signal>clicked()</signal>
   <receiver>m_pWikipediaBrowser</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>98</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel" >
     <x>106</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pWikipediaBrowser</sender>
   <signal>forwardAvailable(bool)</signal>
   <receiver>m_pForwardButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>135</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel" >
     <x>98</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pWikipediaBrowser</sender>
   <signal>backwardAvailable(bool)</signal>
   <receiver>m_pBackButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>39</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel" >
     <x>32</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
